src/components/ebay-button/index.marko

```diff
@@ -49,11 +49,21 @@ $ {
         !truncateClass && !fixedHeightClass && sizeClass,
         (priority === "secondary" ||
             priority === "primary" ||
-            priority === "delete") &&
+            priority === "delete" ||
+            priority === "tertiary") &&
```

Comment:
Lets change this since there are too many checks imo

Something like:
```
const validPriorities = ['secondary', 'primary', 'delete', 'tertiary'];
```

And then here:
```
validPriorities.contains(priority) && `${baseClass}--${priority}`
```
(Or you can do indexOf)